testpmd/hooks/roles/apply-cnf-cert-config/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+app_ns: "{{ dci_openshift_app_ns|default('example-cnf') }}"
+operators_regexp: "{{ tnf_operators_regexp|default('cnf-app-mac|testpmd-operator') }}"
```

Comment:
Yes, I've just included it in my last commits, because I was studying how to do it. The regex to be applied to the operators will be based on the name, in their case.